Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'govuk_design_system_formbuilder'
 gem 'jquery-rails'
 gem 'pg', '~> 1.1'
 gem 'puma'
-gem 'rails', '~> 5.2.4'
+gem 'rails', '~> 5.2.4.4'
```

Comment:
Doing this will restrict future bumps as this is reducing minor releases to the last .x version (so for example, if tomorrow version 5.2.5 is released, this will forbid the update despite still being considered a minor release).

We want any version in the 5.2 branch thus the `~> 5.2.4` so it considers any minor released in the 5.2.x.x range.

So no need to change this really, just doing a `bundle update rails` will figure it out automatically which version it can be updated and apply the changes to the lock file.